custom_components/tuya_local/devices/holman_wx8_irrigation_controller.yaml

```diff
@@ -0,0 +1,240 @@
+name: Holman WX8
```

Comment:
The top level name should be generic and describe the purpose of the device (like "Irrigation controller").  The reason is that many of these Chinese made devices are often sold under different brand names in different countries.